_posts/news/2019-06-03-announcing-scalajs-1.0.0-M8.md

```diff
@@ -0,0 +1,525 @@
+---
+layout: post
+title: Announcing Scala.js 1.0.0-M8
+category: news
+tags: [releases]
+permalink: /news/2019/06/03/announcing-scalajs-1.0.0-M8/
+---
+
+
+We are very excited to announce the release of Scala.js 1.0.0-M8!
+
+This development release is mostly intended for testing purposes, and as a synchronization point with library authors so that they can start upgrading in preparation for the final release.
+
+As the change in "major" version number witnesses, this release is *not* binary compatible with 0.6.x, nor with the previous milestones of the 1.x series.
+Libraries need to be recompiled and republished using this milestone to be compatible.
+
+Moreover, this release is not entirely source compatible with 0.6.x either.
+We expect, however, that further milestones for 1.0.0 will stay backward source compatible with this milestone.
+
+These release notes contain cumulative changes with respect to 0.6.28.
+Compared to 1.0.0-M7, the following changes are noteworthy:
+
+* Add support for Scala 2.13.0-RC3, and hopefully for Scala 2.13.0 final
+* Overall bring up-to-date with improvements and bug fixes available in 0.6.28
+* Scala.js now emits ECMAScript 2015 code by default
+* The I/O of the linker API and the `JSEnv` API have been refactored to be asynchronous in nature, completing the refactoring of the linker API itself which was done in 1.0.0-M7
```

Comment:
Consider making it clearer here that there will still be more changes:
- Do not mention `JSEnv` at all? It is not async. Or mention as a second bullet that it is independent of I/O now.
- Maybe just say something like: "The linker API is now fully asynchronous. Clean separation in different compatibility layers is pending (#3656)".

Reply:
Done.